source/languages/en/riak/dev/search/search-data-types.md

```diff
@@ -1406,17 +1611,27 @@ map:
 String query = "alter_ego_map.name_register:*";
 
 // Again using the same method from above:
-int numberFound = results.numResults(); // 1
+int numberFound = results.numResults(); // 2
 ```
 
 ```ruby
 results = client.search('customers', 'alter_ego_map.name_register:*')
-results['num_found'] # 1
+results['num_found'] # 2
+```
+
+```php
+$response = (new \Basho\Riak\Command\Builder\Search\FetchObjects($riak))
+  ->withIndexName('customers')
+  ->withQuery('alter_ego_map.name_register:*')
+  ->build()
+  ->execute();
+
+$response->getNumFound(); // 2
 ```
 
 ```python
 results = client.fulltext_search('customers', 'alter_ego_map.name_register:*')
-results['num_found'] # 1
+results['num_found'] # 2
```

Comment:
Nah, just confirm that this result is consistent with the previous examples building up to it. I think by setting the alter_ego submap for both elba and jett, a lookup on name_register:* would return 2 results.